airbyte-cdk/python/unit_tests/connector_builder/test_message_grouper.py

```diff
@@ -500,7 +501,7 @@ def test_get_grouped_messages_with_many_slices(mock_entrypoint_read: Mock) -> No
     assert len(stream_read.slices[1].pages[1].records) == 1
     assert len(stream_read.slices[1].pages[2].records) == 0
 
-    assert stream_read.slices[1].state[0].stream.stream_state == {"a_timestamp": 123}
+    assert stream_read.slices[1].state[0].stream.stream_state == AirbyteStateBlob(a_timestamp=123)
```

Comment:
Test equality assertion updates: `Models are no longer equal to the dicts containing their data.`

<img width="720" alt="image" src="https://github.com/airbytehq/airbyte/assets/22333512/2abe2d0b-1ac3-4dbb-946a-49f5ee5fdfd2">